assembl/static2/js/app/components/debate/multiColumns/columnsView.jsx

```diff
@@ -70,7 +98,7 @@ class ColumnsView extends React.Component {
                     style={isColumnViewInline ? { width: `${MIN_WIDTH_COLUMN}px` } : {}}
                   >
                     {isSynthesis
-                      ? <div id={`synthesis-${classifier}`} className="box synthesis">
+                      ? <div id={synthesisId} className="box synthesis">
```

Comment:
Why does the Synthesis have an `id` instead of a `class`?

Reply:
to be able to scroll down to it

Reply:
Ah okay.